go/tasks/plugins/svc/service_client.go

```diff
@@ -0,0 +1,15 @@
+package svc
+
+import (
+	"context"
+)
+
+type CommandStatus string
+
+//go:generate mockery -all -case=snake
+
+type ServiceClient interface {
```

Comment:
docs

Reply:
Sorry, can you clarify what you mean by docs? Do you mean to add comments to this file which will serve as docs? Or are there other docs I should be updating?

Reply:
ah I meant godocs so yeah comments on the public types, fields and funcs... ideally also a package level comment to explain what this is